app/facades/quote_facade.rb

```diff
@@ -0,0 +1,17 @@
+class QuoteFacade
+  def self.quote_search(quote)
```

Comment:
So since we are wanting to get both the `:totalCount` and the `quote attributes` and only make a single call to our api, we can pass two things. It could look something like this:
```
def self.quote_search(quote)
   results = service.get_quotes_search(quote)
   quotes = results[:results].map do |attributes|
      Quote.new(attributes)
   end
   [results[:totalCount], quotes]
end
```
Now we would have access to both total_count and quote attributes for our controller.